start

```diff
@@ -577,7 +577,7 @@ function stellar_core_status() {
 function soroban_rpc_status() {
   echo "soroban rpc: waiting for ready state..."
   COUNTER=1
-  while ! $(curl --silent --location --request POST 'http://localhost:8003/soroban/rpc' \
+  while ! $(curl --silent --location --request POST 'http://localhost:8000/soroban/rpc' \
```

Comment:
Isn't the 8000 is the proxy ( nginx ) whereas the 8003 is the actual rpc ? also, you might want to drop the `/soroban/rpc` and see if that works.
```
                location /soroban/rpc {
                        rewrite /soroban/rpc / break;
                        proxy_set_header Host $http_host;
                        proxy_pass http://127.0.0.1:8003;
                        proxy_redirect off;
                }
```

Reply:
Yes. I figured nginx port would be safer (It's closer to what the user would do), but apparently there've been some connectivity issues with that. Will update this PR.

Reply:
I think that testing the nginx is also worth doing - but it's a different task to tackle.